### ISSUE TITLE
Support for Notebook resources

### ISSUE THREAD

**Issue by taiidani** [terraform-providers/terraform-provider-datadog#261]
Feature request for programmatically creating Notebooks for https://app.datadoghq.com/notebook/list. My use case is that my team uses Notebooks to document our operational runbooks, and having those in the code alongside the applications (as well as the Monitors hyperlinking to the runbooks) would be a pleasant combination.

Current state appears to be that no backend API support is present on the Datadog side so it looks like this will be impossible to implement. However I'd like to have this issue on record to identify if there's a demand for it.

**Comment by Skyfallz** [comment]
Hi,

notebooks are awesome for diagnostic purposes, but the fact that we can't Terraform it makes it difficult to use in a CI/CD context.

Any update/milestone for this feature @gzussa ?

Thanks,

**Comment by kamal-mahmassani-cko** [comment]
Also interested in this

**Comment by vdahmane** [comment]
Any update for this feature?

**Comment by granular-ryanbonham** [comment]
@gzussa It looks like the DD API now support notebooks. We would also love to see TF support added.

**Comment by gzussa** [comment]
Hi, thanks for the interest in this feature. I am not part of the team in charge of this project anymore. @therve may help provide an ETA or request community help (we love contributor PRs 😃 ).

**Comment by therve** [comment]
I reached out the notebook team, and they'll come around to look at it but can't provide a timeline for now. Thanks.

**Comment by kderck** [comment]
Another vote for notebooks.

**Comment by kderck** [comment]
I can see there is a PR - Any chances of this being merged?

**Comment by btjfsu** [comment]
Also waiting on this change to be merged as well...

**Comment by skarimo** [comment]
Hi, we are still discussing this internally with the api owners and will reach follow up as soon as we have an update. Thanks!

**Comment by koushik-goswami** [comment]
Is there any progress on this?

**Comment by nursultanturdaliev** [comment]
Would be great to have this feature!

**Comment by falkorichter** [comment]
We would also love this to happen, how can one support https://github.com/DataDog/terraform-provider-datadog/pull/1475 to be merged?

**Comment by shenault** [comment]
Hello @therve! Do you have any updates on this? We would like to use this feature too 😃

**Comment by robwell57** [comment]
Would also love this feature to be merged.
This would be an awesome feature to add. Come on [#1475](https://github.com/DataDog/terraform-provider-datadog/pull/1475)

**Comment by russwion** [comment]
@therve Any update?  We need this feature in TF.

**Comment by therve** [comment]
No update for this, sorry.

**Comment by eashweragero101** [comment]
When will the feature release be available

**Comment by eashweragero101** [comment]
Greetings and happy new year! We eagerly await an ETA as we are Kindly specify whether you would like me to rephrase the text in a "operational" style to enhance clarity and rectify any typographical, grammatical, or punctuation mistakes. driven group, this release will enable us to maximize our use of Datadog.

**Comment by JordanHLee** [comment]
@skarimo Any update on when we could see #1475 merged?

**Comment by TheJStone** [comment]
Would love to see this feature merged. Especially as datadog takes a stronger foothold in the incident response space having github based runbooks that can sync into datadog would be great.

**Comment by jean-mercier-hivebrite** [comment]
After 6 years where we are on this ? @Supam

**Comment by giovannarbr** [comment]
Hi! Any updates on this?

**Comment by mekstrem** [comment]
Hello, can you please provide status update on this? @Supam

**Comment by adamthewilliam** [comment]
This feature request was raised 6 years ago and still no movement. What a joke. How difficult is it to create a new DataDog resource for notebooks lol

**Comment by fabn** [comment]
Hey there, any news. It would be super useful to have this.

**Comment by nguyenanhducs** [comment]
Any update?

**Comment by eashweragero101** [comment]
Dyam Datadog is just dandy, but they take a year to resolve a feature

**Comment by drAlberT** [comment]
Any chance to have this addressed?

**Comment by igor-mardari-sonarsource** [comment]
Interested into this

**Comment by philBatardiere** [comment]
Interested into this
